anomalib/models/cflow/config.yaml

```diff
@@ -49,8 +49,11 @@ metrics:
 project:
   seed: 0
   path: ./results
-  log_images_to: [local]
-  logger: false # options: [tensorboard, wandb, csv] or combinations.
+
+logging:
+  log_images_to: ["local"] # options: [wandb, tensorboard, local]. Make sure you also set logger with using wandb or tensorboard.
```

Comment:
I think we should decide how to handle `log_images_to` parameter before merging this PR. It could be annoying to the user if we decide to change it again in the future. @djdameln, @ashwinvaidya17

Reply:
I think it will take some time to change this. We'll need a design first and then implement the changes. So I would suggest to merge this first to prevent delays.